src/api/draftSpecs.ts

```diff
@@ -0,0 +1,47 @@
+import { callSupabase, TABLES } from 'services/supabase';
+
+export const createDraftSpec = (
+    draftId: string | null,
+    catalogName: string,
+    specType: 'materialization',
+    draftSpec: any
+) => {
+    return callSupabase(TABLES.DRAFT_SPECS, [
+        {
+            draft_id: draftId,
+            catalog_name: catalogName,
+            spec_type: specType,
+            spec: draftSpec,
+        },
+    ]);
+};
+
+export const generateDraftSpec = (
```

Comment:
Not just including the calls in here but also helper functions closely related to the calls.